wikitextprocessor/core.py

```diff
@@ -16,7 +16,7 @@
 from collections.abc import Sequence
 from dataclasses import dataclass
 from functools import lru_cache
-from importlib.resources import files
+from importlib_resources import files
```

Comment:
You should check the python version(maybe using the `platform` library) and only use the backport package for older Python version.

Reply:
Any reason why not to use ```if sys.version_info < (3, 10):```?